table/tables/tables.go

```diff
@@ -862,9 +842,9 @@ func (t *TableCommon) addIndices(sctx sessionctx.Context, recordID kv.Handle, r
 }
 
 // RowWithCols implements table.Table RowWithCols interface.
```

Comment:
```suggestion
// RowWithCols is used to get the corresponding column datum values with the given handle.
```

Reply:
The memory table is still using this function, for the data from `infoschema`

Reply:
But `table.Table` does not have the method `RowWithCols` anymore.

Reply:
Yes, we can, for example, compose the interfaces:

```
type Table interface {...}

type MemoryTable interface {
        Table
        RowWithCols()
}
````

And we can do that step by step after this refactor ...